hack/build-images.sh

```diff
@@ -120,6 +123,27 @@ image openshift/hello-openshift              examples/hello-openshift
 docker build --no-cache -t openshift/deployment-example:v1 examples/deployment
 docker build --no-cache -t openshift/deployment-example:v2 -f examples/deployment/Dockerfile.v2 examples/deployment
 
+# images for integrated services (e.g. logging and metrics)
+clonedir="$(mktemp -d 2>/dev/null || mktemp -d -t clonedir.XXXXXX)"
+pushd $clonedir
+# logging
+git clone --depth 1 --branch ${OS_LOGGING_BRANCH} https://github.com/openshift/origin-aggregated-logging.git 
+image openshift/origin-logging-curator        origin-aggregated-logging/curator
+image openshift/origin-logging-deployment     origin-aggregated-logging/deployment
+image openshift/origin-logging-elasticsearch  origin-aggregated-logging/elasticsearch
+image openshift/origin-logging-fluentd        origin-aggregated-logging/fluentd
+image openshift/origin-logging-kibana         origin-aggregated-logging/kibana
```

Comment:
Cleaning my machine of all base images, etc, it took approximately 31 minutes to build all logging and metrics images.  This is on a T530 with 8G of RAM